integrationtest/mapper/orm-coordination-outbox-polling/src/test/java/org/hibernate/search/integrationtest/mapper/orm/coordination/outboxpolling/FilteringOutboxEventFinder.java

```diff
@@ -99,15 +100,15 @@ public synchronized void hideAllEvents() {
 	public List<OutboxEvent> findOutboxEventsNoFilter(Session session) {
 		checkFiltering();
 		Query<OutboxEvent> query = session.createQuery(
-				"select e from " + ENTITY_NAME + " e order by e.id", OutboxEvent.class );
+				"select e from " + ENTITY_NAME + " e order by e.created, e.id", OutboxEvent.class );
```

Comment:
Ha. I just remembered that we used to do that in the past, but we stopped because timestamp precision wasn't great in some databases (millisecond).

That's not a problem in real-world applications as we don't need the ordering to be exact, but it **is** a problem in tests where two subsequent events might end up with the same created timestamp and thus end up with an undefined relative order...

Let's run tests multiple times on each database before we merge this, because we might have unpleasant surprises (flaky tests).